mystique/build.gradle

```diff
@@ -186,14 +194,14 @@ tasks.create(name: "s3Uploader") << {
         AmazonS3 s3 = new AmazonS3Client(awsCreds)
         TransferManager tx = new TransferManager(awsCreds)
         // push maven artifacts
-        def upload = tx.uploadDirectory("release", "dui", new File("/tmp/_mvn_dui"), true)
+        def upload = tx.uploadDirectory("dui-release", "dui", new File("/tmp/_mvn_dui"), true)
```

Comment:
Revert this to `release` and maybe use environment variables or local configs to override this. Also do this for all the changes for the bucket name.